docs/graphql/core/actions/clean-up-actions-logs.rst

```diff
@@ -0,0 +1,47 @@
+.. meta::
+   :description: clean up actions logs
+   :keywords: hasura, docs, actions, clean up, async actions
+
+.. _clean_up_actions_logs:
+
+Clean up actions logs
+=====================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Introduction
+------------
+Hasura stores actions logs from asynchronous operations. If there are a lot of logs, the tables can get huge and you may want to prune them.
+You can use any of the following options to prune your logs depending on your need.
+
+The table involved
+------------------
+
+Actions has a table managed by Hasura:
+
+- ``hdb_catalog.hdb_action_log``: This is the table that stores all captured logs.
+
+Option 1: Get all logs
+----------------------
+.. code-block:: SQL
+   
+   SELECT * FROM hdb_catalog.hdb_action_log;
+
+Option 2: Delete one log
+------------------------
+.. code-block:: SQL
+   
+   DELETE FROM hdb_catalog.hdb_action_log WHERE id = '<action-id>';
+
+Option 3: Clear all logs
+------------------------
+.. admonition:: Warning
+ 
+   This will clear all captured logs.
```

Comment:
This we already say in the title. If the action is irreversible, we can do a note saying "Clearing all logs is irreversible". But before we should check if this is actually the case.

Reply:
Also, we can add this note after the command

Reply:
Yes, it is irreversible, so I will change the statement. @marionschleifer